pay-api/src/pay_api/utils/trace.py

```diff
@@ -12,11 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Bring in the Tracer."""
-from sbc_common_components.tracing.api_tracer import ApiTracer
-from sbc_common_components.tracing.api_tracing import ApiTracing
+# from sbc_common_components.tracing.api_tracer import ApiTracer
+# from sbc_common_components.tracing.api_tracing import ApiTracing
 
 
 # initialize tracer
-API_TRACER = ApiTracer('Payment Services')
-tracing = ApiTracing(  # pylint: disable=invalid-name; lower case name as used by convention in most Flask apps
-    API_TRACER.tracer)
+# TODO Fix this later? not sure if it's even used.
```

Comment:
Remove tracers, have to check with Patrick if they're used

Reply:
He says they aren't used, but we might use in the future, I'll see if I can limp this along for now